source/common/redis/conn_pool_impl.cc

```diff
@@ -125,6 +131,7 @@ void ClientImpl::onRespValue(RespValuePtr&& value) {
   ASSERT(!pending_requests_.empty());
   PendingRequest& request = pending_requests_.front();
   if (!request.canceled_) {
+    host_->outlierDetector().putHttpResponseCode(enumToInt(Http::Code::OK));
```

Comment:
IMO even if the request was cancelled by downstream for whatever reason, it still succeeded for the upstream, and we should count it as a success? Thoughts?